### PR TITLE
Resolve suggestion

### LINKED ISSUE #803: resolve / reinstall allが必要な状態を検知する
### Summary / 概要

resolve / reinstall allが必要な状態を検知する

### Purpose / 目的

resolve / reinstall allが必要な状態を検知してsuggestする

### Additional context / 補足

_No response_

### PR DESCRIPTION
**Pull request by anatawa12**
Fixes #803

### CODE REVIEW

File: vrc-get-gui/locales/en.json5 [hunk 1]
```diff
@@ -94,6 +94,8 @@
     "projects:manage:button:select repositories": "Select Repositories",
     "vpm repositories:source:official": "Official",
     "vpm repositories:source:curated": "Curated",
+    "projects:manage:suggest resolve": "Your project does not installed some packages required by the project. It's highly recommended to resolve packages.",
```
Comment (Sayamame-beans) [inline vrc-get-gui/locales/en.json5:97]: "does not installed" 文法が

Comment (anatawa12) [reply vrc-get-gui/locales/en.json5:97]:
```suggestion
    "projects:manage:suggest resolve": "Some packages required by the project are not installed. It's highly recommended to resolve packages.",
```

かな

Comment (Sayamame-beans) [reply vrc-get-gui/locales/en.json5:97]: よさそう

**Review by Sayamame-beans — APPROVED** [review]
多分OK!

**Comment by anatawa12** [comment]
For translators: This message is very rare to show to user so I'll keep english for ALCOM 0.1.1 release.
(I'm hurrying to release 0.1.1

**Pull request merged**